src/DETHRACE/common/controls.c

```diff
@@ -96,207 +96,256 @@ char* gEdit_mode_names[10] = {
+    {
+        // eEdit_mode_depth
+        { 0 }, // F5
+        { 0 }, // F6
+        { 0 }, // F7
+        { 0 }, // F8
+        { 0 }, // F10
+        { 0 }, // F11
+        { 0 }, // F12
+        { 0 }, // 0
+        { 0 }, // 1
+        { 0 }, // 2
+        { 0 }, // 3
+        { 0 }, // 4
+        { 0 }, // 5
+        { 0 }, // 6
+        { 0 }, // 7
+        { 0 }, // 8
+        { 0 }, // 9
+        { 0 }, // not used
     },
-    {  // eEdit_mode_damage
-        { 0 },  // F5
-        { 0 },  // F6
-        { 0 },  // F7
-        { 0 },  // F8
-        { 0 },  // F10
-        { 0 },  // F11
-        { 0 },  // F12
-        { 0 },  // 0
-        { 0 },  // 1
-        { 0 },  // 2
-        { 0 },  // 3
-        { 0 },  // 4
-        { 0 },  // 5
-        { 0 },  // 6
-        { 0 },  // 7
-        { 0 },  // 8
-        { 0 },  // 9
-        { 0 },  // not used
+    {
+        // eEdit_mode_damage
+        { 0 }, // F5
+        { 0 }, // F6
+        { 0 }, // F7
+        { 0 }, // F8
+        { 0 }, // F10
+        { 0 }, // F11
+        { 0 }, // F12
+        { 0 }, // 0
+        { 0 }, // 1
+        { 0 }, // 2
+        { 0 }, // 3
+        { 0 }, // 4
+        { 0 }, // 5
+        { 0 }, // 6
+        { 0 }, // 7
+        { 0 }, // 8
+        { 0 }, // 9
+        { 0 }, // not used
     },
-    {  // eEdit_mode_bonnet
-        { 0 },  // F5
-        { 0 },  // F6
-        { 0 },  // F7
-        { 0 },  // F8
-        { 0 },  // F10
-        { 0 },  // F11
-        { 0 },  // F12
-        { 0 },  // 0
-        { 0 },  // 1
-        { 0 },  // 2
-        { 0 },  // 3
-        { 0 },  // 4
-        { 0 },  // 5
-        { 0 },  // 6
-        { 0 },  // 7
-        { 0 },  // 8
-        { 0 },  // 9
-        { 0 },  // not used
+    {
+        // eEdit_mode_bonnet
+        { 0 }, // F5
+        { 0 }, // F6
+        { 0 }, // F7
+        { 0 }, // F8
+        { 0 }, // F10
+        { 0 }, // F11
+        { 0 }, // F12
+        { 0 }, // 0
+        { 0 }, // 1
+        { 0 }, // 2
+        { 0 }, // 3
+        { 0 }, // 4
+        { 0 }, // 5
+        { 0 }, // 6
+        { 0 }, // 7
+        { 0 }, // 8
+        { 0 }, // 9
+        { 0 }, // not used
     },
-    {  // eEdit_mode_options
-        { 0 },  // F5
-        { 0 },  // F6
-        { 0 },  // F7
-        { 0 },  // F8
-        { 0 },  // F10
-        { 0 },  // F11
-        { 0 },  // F12
-        { 0 },  // 0
-        { CycleCarSimplificationLevel, NULL, NULL, NULL, CycleCarTexturingLevel, },  // 1
-        { ToggleShadow, NULL, NULL, NULL, ToggleSmoke, },  // 2
-        { CycleWallTexturingLevel, NULL, NULL, NULL, CycleRoadTexturingLevel, },  // 3
-        { ToggleSky, NULL, NULL, NULL, ToggleDepthCueing, },  // 4
-        { CycleYonFactor, NULL, NULL, NULL, ToggleAccessoryRendering, },  // 5
-        { DecreaseYon, NULL, NULL, NULL, IncreaseYon,  },  // 6
-        { CycleSoundDetailLevel,  },  // 7
-        { 0 },  // 8
-        { 0 },  // 9
-        { 0 },  // not used
+    {
```

Comment:
@madebr does this auto-format change look ok to you? I have `clang-format` set up to run automatically on my system

Reply:
This looks fine to me!